### DISCUSSION TITLE
How to make the call to tts_to_file() less verbose

### DISCUSSION THREAD

**Discussion by vincepmartin** [coqui-ai/TTS#2372] [General Q&A] [answered]
Hi everyone,

I am using the TTS to convert text to speech, it works quite well, however, I am having an issue where this call is quite verbose.  Pages of messages get generated.  Is there a way to dial this back?

The command I am using via python is 
`self.tts.tts_to_file(text=doc, file_path=filePath, speaker=self.tts.speakers[0], language=self.tts.languages[0])`

Generates output as such

![image](https://user-images.githubusercontent.com/4640602/222258498-de9e9065-7888-4f1a-8c31-232f11f41890.png)

**Comment by erogol** [comment, marked as answer]
Currently, there is no way to set the verbosity level, unfortunately.

**Reply by vincepmartin** [reply]
Thanks.  I appreciate you getting back to me!

**Comment by eginhard** [comment]
In our fork we've now switched uses of `print` to use standard Python logging, allowing for configurable output: https://github.com/idiap/coqui-ai-TTS
